File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/weather/model/Forecast.java

package restaurant.sa.com.sarestaurant.appview.weather.model;

import com.google.gson.annotations.SerializedName;
public class Forecast {

    @SerializedName("code")
    private String mCode;
    @SerializedName("date")
    private String mDate;
    @SerializedName("day")
    private String mDay;
    @SerializedName("high")
    private String mHigh;
    @SerializedName("low")
    private String mLow;
    @SerializedName("text")
    private String mText;

    public String getCode() {
        return mCode;
    }

    public void setCode(String code) {
        mCode = code;
    }

    public String getDate() {
        return mDate;
    }

    public void setDate(String date) {
        mDate = date;
    }

    public String getDay() {
        return mDay;
    }

    public void setDay(String day) {
        mDay = day;
    }

    public String getHigh() {
        return mHigh;
    }

    public void setHigh(String high) {
        mHigh = high;
    }

    public String getLow() {
        return mLow;
    }

    public void setLow(String low) {
        mLow = low;
    }

    public String getText() {
        return mText;
    }

    public void setText(String text) {
        mText = text;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/weather/model/Forecast.kt
package restaurant.sa.com.sarestaurant.appview.weather.model

import com.google.gson.annotations.SerializedName

class Forecast {

    @SerializedName("code")
    var code: String? = null
    @SerializedName("date")
    var date: String? = null
    @SerializedName("day")
    var day: String? = null
    @SerializedName("high")
    var high: String? = null
    @SerializedName("low")
    var low: String? = null
    @SerializedName("text")
    var text: String? = null

}
